cogs/recruit.py

```diff
@@ -1,320 +1,66 @@
-            Choice(name="remove", value="remove")
-        ]
-    )
-    async def exceptions(self, ctx: commands.Context, action: str, region: str):
-        await ctx.defer()
-
-        region = region.lower().replace(" ", "_")
-
-        if action == "add":
-            if region in configInstance.data.recruitment_exceptions:
-                await ctx.reply("Region already in exceptions.")
-            else:
-                configInstance.data.recruitment_exceptions.append(region)
-                await ctx.reply("Region added to exceptions.")
-        elif action == "remove":
-            if region not in configInstance.data.recruitment_exceptions:
-                await ctx.reply("Region not in exceptions.")
-            else:
-                configInstance.data.recruitment_exceptions.remove(region)
-                await ctx.reply("Region removed from exceptions.")
-
-        configInstance.writeConfig()
-
-    @tasks.loop(seconds=configInstance.data.polling_rate)
-    async def newnations_polling(self):
-        current_time = datetime.now()
-
-        while len(self.request_times) >= configInstance.data.period_max:
-            elapsed = (current_time - self.request_times[0]).total_seconds()  # type: ignore
-
-            if elapsed > configInstance.data.period:
-                del self.request_times[0]
-            else:
-                self.bot.std.info(f"Sleeping for {configInstance.data.period - elapsed} seconds")
-                await asyncio.sleep(configInstance.data.period - elapsed)
-
-        self.request_times.append(current_time)
-
-        headers = {
-            "User-Agent": f"Euro Recruitment Bot, developed by upcnationstates@gmail.com, used by {configInstance.data.operator}"}
-
-        try:
-            self.bot.std.info("Polling NEWNATIONS shard.")
-
-            new_nations = bs(
-                requests.get("https://www.nationstates.net/cgi-bin/api.cgi?q=newnationdetails", headers=headers).text,
-                "xml").find_all("NEWNATION")  # type: ignore -- BeautifulSoup returns a variant type.
-        except:
-            # certified error handling moment
-            self.bot.std.error("An unspecified error occurred while trying to poll the NEWNATIONS shard")
-        else:
-
-            current_nations = [nation.name for nation in self.bot.queue.nations]
-            last_update = int(self.bot.queue.last_update.timestamp())
-
-            valid_nations: List[str] = []
-
-            for nation in reversed(new_nations):
-                if nation.attrs["name"] not in current_nations \
-                        and nation.REGION.text not in configInstance.data.recruitment_exceptions \
-                        and int(nation.FOUNDEDTIME.text) > last_update \
-                        and not self.pattern.search(nation.attrs["name"]):
-                    valid_nations.append(nation.attrs["name"])
-
-            self.bot.queue.update(valid_nations)
+    async def init(self, ctx: commands.Context):
```

Comment:
Do we still need this? Can we delete the command?

Reply:
No, we'll need this one to send a message with the new view. It works in a similar way to the last one -- you start the bot, run !init, copy that message ID into settings.json, and then restart and you're good. The view is persistent, so it will be reconnected between restarts, this is just a first time setup thing. There might be a way to attach it to an old message, but I think that that would be more of a hassle than this.